src/snapred/backend/data/LocalDataService.py

```diff
@@ -535,292 +340,155 @@ def writeNormalizationRecord(
         Persists a `NormalizationRecord` to either a new version folder, or overwrites a specific version.
         -- side effect: updates version numbers of incoming `NormalizationRecord` and its nested `Normalization`.
         """
-        runNumber = record.runNumber
-        stateId, _ = self._generateStateId(runNumber)
-        previousVersion = self._getLatestNormalizationVersionNumber(stateId, record.useLiteMode)
-        if version is None:
-            version = max(record.version, previousVersion + 1)
-        recordPath: Path = self.getNormalizationRecordFilePath(runNumber, record.useLiteMode, version)
-        record.version = version
-
-        # There seems no need to write the _nested_ Normalization,
-        # because it's written to a separate file during 'writeNormalizationState'.
-        # However, if it is going to be _nested_, this marks it with the correct version.
-        # (For example, use pydantic Field(exclude=True) to _stop_ nesting it.)
-        record.calibration.version = version
-
-        normalizationPath: Path = self._constructNormalizationDataPath(runNumber, record.useLiteMode, version)
-        # check if directory exists for runId
-        if not normalizationPath.exists():
-            os.makedirs(normalizationPath)
```

Comment:
All of this is now handled by the `Indexor`

Reply:
Good.  **So long as indexer is not called within these write methods!**